src/providers/json-rpc-provider.ts

```diff
@@ -81,11 +82,18 @@ export class JsonRpcProvider extends Provider {
     /**
      * Query for block info from the RPC
      * See [docs for more info](https://docs.nearprotocol.com/docs/interaction/rpc#block)
-     * @param blockId Block hash or height
-     * @returns {Promise<BlockResult>}
```

Comment:
I've deleted these lines because we're using TypeScript & TypeDoc, so these comments provide no information that isn't available via the argument typings. Explaining the new `blockQuery` argument with English seemed harder than just using the typings, so I opted to just remove the comment.

Reply:
agree 100% that we shouldn't have docs which merely restate types

however I think in context of this method it is interesting to understand how finality argument would work, it is less obvious that for block id

Reply:
Yeah, but that is explained in the docs link

I think we should move such docs inline into these comments (which will add them to the generated near-api-js API docs), and slim down the docs.near.org docs, but that seems like a separate issue. And when we do that, that's when I imagine we will move the explanation of the `finality` argument into this comment.